frontend/assets/stylesheets/components/_callout.scss

```diff
@@ -0,0 +1,39 @@
+.call-out-container {
```

Comment:
Picky but could we add a block comment at the top of this file describing what it does, like other components?